Week-13/WeatherApp/utils/response.js

```diff
@@ -0,0 +1,16 @@
+const GenerateSuccessRespone = (body) => {
+    return {
+        "code": 200,
+        "message": "SUCCESS",
+        "data": body
+    }
+}
+const GenerateErrorRespone = (body) => {
+    return {
+        "code": 500,
```

Comment:
use the error code from the weather api

Reply:
The "GenerateErrorRespone" Method returns the JSON output when an Internal Error Occurs within the API (catch). Modified GenerateResponse method to return the Success/Error Code sent by Weather API. Pushed the changes